tests/tensorflow/test_tensorflow2_model_export.py

```diff
@@ -226,6 +203,105 @@ def saved_tf_categorical_model(tmpdir):
     )
 
 
+def save_tf_estimator_model(
+    tf_saved_model_dir, tf_meta_graph_tags, tf_signature_def_key, path, mlflow_model=None
+):
+    """
+    A helper method to save tf estimator model as a mlflow model, it is used for testing loading
+    tf estimator model saved by previous mlflow version.
+    """
+    os.makedirs(path)
```

Comment:
I think we can do:

1. Create an environment with an older version of MLflow.
2. Run code that logs a tensorflow model.